acceptance_tests/utilities/sftp_utility.py

```diff
@@ -43,3 +47,11 @@ def _get_file_lines_as_list(self, file_path):
         with self._sftp_client.open(file_path) as sftp_file:
             content = sftp_file.read().decode('utf-8')
             return content.rstrip().split('\n')
+
+    def get_file_contents_as_string(self, file_path):
+        with self._sftp_client.open(file_path) as sftp_file:
+            content = sftp_file.read().decode('utf-8')
```

Comment:
No real need to store in a variable if it's just going to be immediately returned.

```suggestion
            return sftp_file.read().decode('utf-8')
```